UI cleanup in the auth screen

hackertronix opened the pull request:
@alexsullivan114, I fixed some styling related things in this PR alongside the tweaks to the movie item

hackertronix committed:
UI cleanup in the auth screen

hackertronix committed:
Reformat

filbabic reviewed (changes requested):
Hey!

Can you make sure to use the same formatting as in the project (XML also uses 2 spaces for its ind. so it's more visible if we have to show it in the book).

I think you've formatted to four here. We can all just use the project style and it should be fine!

hackertronix committed:
Reformat as per RW code style

hackertronix commented:
@filbabic fixed! 🙌

hackertronix committed:
Fix z ordering

# Conflicts:
#	Cinematic/app/src/main/res/drawable/drawable_rounded_corner.xml
#	Cinematic/app/src/main/res/layout/fragment_auth.xml
#	Cinematic/app/src/main/res/layout/fragment_login.xml
#	Cinematic/app/src/main/res/layout/fragment_signup.xml
#	Cinematic/app/src/main/res/layout/item_movie.xml
